TombEngine/Game/effects/effects.cpp

```diff
@@ -52,7 +52,7 @@ int SplashCount = 0;
 Vector3i NodeVectors[ParticleNodeOffsetIDs::NodeMax];
 NODEOFFSET_INFO NodeOffsets[ParticleNodeOffsetIDs::NodeMax] =
 {
-	{ -16, 40, 160, -LM_LHAND, false }, // TR5 offset 0, TODO: This mesh is invalid as it can't be negative. -- TokyoSU 23.02.20
+	{ -16, 40, 160, 13, false },		// TR5 offset 0
```

Comment:
Is this bone index for Lara? If it is, it should use enum, otherwise it should not and everything is okay until we demagic enemy bone indices.

Reply:
Node offset is not for lara, i think it was for a entity and was just a typo.